src/View/SSViewer.php

```diff
@@ -134,27 +136,31 @@ class SSViewer implements Flushable
      * List of items being processed
      *
      * @var array
+     * @deprecated 5.4.0 Will be moved to SilverStripe\View\SSTemplateEngine
      */
     protected static $topLevel = [];
```

Comment:
We don't really have a policy about what to do with properties - I've opted to deprecate it but didn't bother mentioning it in the docs